databroker/headersource/shim.py

```diff
@@ -24,7 +24,11 @@ def __call__(self, text_search=None, filters=None, **kwargs):
         else:
             # Include empty {} here so that '$and' gets at least one query.
             _format_time(kwargs, self.mds.config['timezone'])
-            query = {'$and': [{}] + [kwargs] + filters}
+            formatted_filters = []
+            for v in filters:
+                _formatted_time(v, self.mds.config['timezone'])
```

Comment:
I think you want `_format_time` here, not `_formatted_time`. Also, this might be a nice place to use a list comprehension.